apps/ledger-live-mobile/src/const/navigation.ts

```diff
@@ -560,8 +561,8 @@ export enum NavigatorName {
   RequestAccount = "RequestAccount",
   RequestAccountsAddAccounts = "RequestAccountsAddAccounts",
 
-  // Edit transaction
-  EditTransaction = "EditTransaction",
+  // Evm
+  EvmEditTransaction = "EvmEditTransaction",
```

Comment:
Can be move with the rest close to line 333

Reply:
Not really, it's not the same enum, `NavigatorName` vs `ScreenName`